app/controllers/courses/github_repos_controller.rb

```diff
@@ -0,0 +1,37 @@
+# This controller is nested in the routing hierarchy as a child controller of 
+# courses_controller
+# see this tutorial for details on how we do this https://gist.github.com/jhjguxin/3074080
+require 'Octokit_Wrapper'
+
+module Courses
+  class GithubReposController < ApplicationController
+    layout 'courses'
+    before_action :load_parent
+    before_action :set_github_repo, only: [:show]
+
+    load_and_authorize_resource :course
+    load_and_authorize_resource :github_repo, through: :course
+
+    def index
+      @github_repos = @parent.github_repos.all
```

Comment:
This is the non-api controller, right? Do we need to set this? I don't see it being passed to the React component and that's probably not a good idea anyway